src/forwardRef.js

```diff
@@ -0,0 +1,16 @@
+import React, {cloneElement, forwardRef} from 'react';
+import {preserveRef} from './utils';
+
+export default Tippy =>
```

Comment:
maybe let's export a named function for better debugging?

Reply:
I don't think it matters in this case